node/actors/artifacts.go

```diff
@@ -24,24 +28,57 @@ type ArtifactReference struct {
 	Size int
 }
 
+func extractTag(location *url.URL) string {
+	return location.Fragment
```

Comment:
What happened to the code that checks for a :, splits, and returns the tag and if none exists it returns "latest"?

Reply:
Hasn't been written.  I'm going to do a PR for artifacts next. It'll address all the functionality